website_project/django_project/settings.py.orig

```diff
@@ -1,31 +1,30 @@
 """
-Django settings for django_project project.
-
-Generated by 'django-admin startproject' using Django 2.2.12.
-
+Django settings for blah project.
+Generated by 'django-admin startproject' using Django 3.1.5.
 For more information on this file, see
-https://docs.djangoproject.com/en/2.2/topics/settings/
-
+https://docs.djangoproject.com/en/3.1/topics/settings/
 For the full list of settings and their values, see
-https://docs.djangoproject.com/en/2.2/ref/settings/
+https://docs.djangoproject.com/en/3.1/ref/settings/
 """
 
+from pathlib import Path
 import os
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+TEMPLATE_DIR = os.path.join(BASE_DIR, 'templates')
 
 
 # Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'z%0z_$&)3cz=+fbb22afkw_dt)lajca%tq*zfj08l^pu_e6-6h'
+SECRET_KEY = 'he(-a3@6*yt&o4%k&m6up8zuk4n3=bk50&q6#)tvfo$3r&$&rh'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
Is this a Secret Key I see :eyes:

Reply:
That secret key will only be valid for my individual local copy, so it shouldn't cause any problems. This file is independent of the actual program, it's being used simply as a reference for the other local copies pulling down from remote repo.